readthedocs/config/config.py

```diff
@@ -159,7 +152,6 @@ class BuildConfigBase:
         'submodules',
     ]
 
-    default_build_image = DOCKER_DEFAULT_VERSION
```

Comment:
I think this one can stay

Reply:
Different configuration files can have a different default image

Reply:
Okay